README.md

```diff
@@ -280,8 +352,8 @@ To test using Docker's `busybox` image follow these steps:
 mkdir rootfs
 tar -C rootfs -xf busybox.tar
 ```
-* Create a file called `config.json` using the example from above.  You can also
-generate a spec using `runc spec`, redirecting the output into `config.json`
+* Create `config.json` and `runtime.json` using the example from above.  You can also
+generate a spec using `runc spec`, which will create those files for you.
```

Comment:
This, on the other hand, is runC-specific information that should stay in this repository.